vizro-core/src/vizro/models/_page.py

```diff
@@ -66,8 +67,17 @@ class Page(VizroBaseModel):
     # TODO[mypy], see: https://github.com/pydantic/pydantic/issues/156 for components field
     components: conlist(Annotated[ComponentType, BeforeValidator(check_captured_callable_model)], min_length=1)  # type: ignore[valid-type]
     title: str = Field(description="Title of the `Page`")
-    description: str = Field(default="", description="Description for meta tags.")
     layout: Annotated[Optional[LayoutType], AfterValidator(set_layout), Field(default=None, validate_default=True)]
+    # TODO: ideally description would have json_schema_input_type=Union[str, Tooltip] attached to the BeforeValidator,
+    #  but this requires pydantic >= 2.9.
+    description: Annotated[
+        Optional[Tooltip],
+        BeforeValidator(coerce_str_to_tooltip),
+        Field(
+            default=None,
+            description="Optional markdown string that adds an icon next to the title.",
```

Comment:
Sorry, if it wasn't clear, but I meant adding that sentence on top of also including the metatag description.
```suggestion
            description="Optional markdown string that adds an icon next to the title.
            Hovering over the icon shows a tooltip with the provided description. This text also serves as the page’s meta description. Defaults to `None`."
```

Reply:
Done in [ad6ca8c](https://github.com/mckinsey/vizro/pull/1144/commits/ad6ca8c844e590dc940bbf9fd73f891f5972f593)